fleiss.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name          = 'fleiss'
-  s.version       = '0.4.4'
+  s.version       = '0.4.5'
```

Comment:
No back-compatible changes. This is a new, optional feature to be explicitly opted-in by setting `FLEISS_LOCK_TTL`.